### PR TITLE
chore: add volunteer maityamit

### PR DESCRIPTION
**Pull request by maityamit**
Add new volunteer Amit Maity

```Android Developer```

### CODE REVIEW

File: data/users/AmitMaity.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "bio": "Android Developer",
+    "image": "https://maityamit.github.io/maityamit-portfolio/assets/images/hero.png",
+    "categories": [
+      {
+        "slug": "mobile-development",
+        "topics": ["android"]
+      }
+    ],
+    "contacts": {
+      "twitter": "https://twitter.com/_maityamit"
```
Comment (codewithvoid) [inline data/users/AmitMaity.json:12]: just username here
```json
     "twitter": "_maityamit"
```

**Comment by codewithvoid** [comment]
Thanks for volunteering @maityamit. Left small comment.

**Comment by maityamit** [comment]
done @codewithvoid 
checkout

**Review by codewithvoid — APPROVED** [review]
Neat. Welcome!

**Pull request merged**